examples/src/discovery.cpp

```diff
@@ -147,7 +147,8 @@ void Discovery::onEntityOnline(la::avdecc::controller::Controller const* const /
 			auto const& obj = objIt.second;
 			if (obj.staticModel->memoryObjectType == la::avdecc::entity::model::MemoryObjectType::PngEntity)
 			{
-				_controller->readDeviceMemory(entity->getEntity().getEntityID(), obj.staticModel->startAddress, obj.staticModel->maximumLength,
+				_controller->readDeviceMemory(
```

Comment:
I haven't tested a new version of clang-format, but is this the formatting result using your version?
I don't like the parameters starting on the next line if this is the case. I won't be able to update clang-format version if there is no option to fix this.

Reply:
yep, this is the default clang-format result. Maybe need to tweak some more options

Reply:
I guess it has wrapped the value here just because of line length limitation. If not, this call will hit 80 symbols threshold. Is this a single place in the code that don't satisfy you? Can you live with it?

![image](https://user-images.githubusercontent.com/1313542/75338461-c9636480-589f-11ea-9f25-2d3ebaffde98.png)

Reply:
I've set the limit to 800 characters because I'd rather have the text editor do the wrapping instead of hardcoding it. This way each developer can choose it's own _virtual_ wrapping.
I don't know if it's the only place in the code as I couldn't find the time to use the latest clang-format and see if it breaks elsewhere.
My lambda patch (which is an improvement of the one that just got integrated into clang-format) fixes that specific formatting _issue_. I don't know why it was not integrated as the behaviour is not consistent by default.

Reply:
Take a look at [clang-format 11 options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)

Reply:
@christophe-calmejane I can clearly tell that this is a single place in the code, because I've lint all the files with the latest `clang-format`

Reply:
I unfortunately won't have the time to check all the new options of cf11 right now.
Can you run the test cases I provided in this post: https://reviews.llvm.org/D44609 against your cf version and post the result here, so I can compare with my version. Thanks.